faqs/README.md

```diff
@@ -144,10 +144,12 @@ The following prices are exclusive of any local taxes and shipping/handling fees
 
 ### What do I get when I buy one?
 
-You get the Raspberry Pi board itself. The [official power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) and an SD card are not included, but they can be purchased at the same time from most places that sell the Raspberry Pi. You can also purchase pre-loaded SD cards; we recommend buying these from us or our licensed distributors rather than from third parties, as our software is updated regularly and cards sold by third parties can quickly become outdated.
+Raspberry Pi devices are available as just the board, or as part of a kit. When buying just the device, you get the Raspberry Pi board itself. The [official power supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) and an SD card are not included, but they can be purchased at the same time from most places that sell the Raspberry Pi. You can also purchase pre-loaded SD cards; we recommend buying these from us or our licensed distributors rather than from third parties, as our software is updated regularly and cards sold by third parties can quickly become outdated.
```

Comment:
> we recommend buying these from us

Is this referring to the Raspberry Pi store in Cambridge, or something else? Maybe this phrase needs updating?
IIRC the [Swag Store](https://swag.raspberrypi.org/) used to sell SD cards, but this is no longer the case (and there's also no links from https://www.raspberrypi.org/ to the Swag Store now).

Reply:
Not sure.  Let me have a think whilst walking the dog.

Reply:
Not sure about the use of `just`. What about:

`Raspberry Pi computers are available separately, or as part of a kit which contain everything you need to get started including a mouse, HDMI cable, SD card, power supply, as well as our official Beginners Guide book. The Raspberry Pi 400 computer contains a case and keyboard built in: other kits come with a separate case and keyboard.`

And for the power supply:

`We recommend using an official Raspberry Pi Power Supply. For the Pi Zero, 1, 2 and 3 use the [Raspberry Pi 1, 2 and 3 Power Supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/), for the Pi 4B and Pi 400 use the [Raspberry Pi USB-C Power Supply](https://www.raspberrypi.org/products/type-c-power-supply/)`.